simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -2,7 +2,7 @@
   <param name="simulate" value="true" />
```

Comment:
Don't worry - I think I wrote that line. /simulate is sub-specific and tells the thruster driver to behave differently. Gazebo automatically sets use_sim_time